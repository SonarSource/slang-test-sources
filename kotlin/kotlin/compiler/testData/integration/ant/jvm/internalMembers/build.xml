<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>

    <target name="build">
        <kotlinc src="${test.data}/module1.kt" output="${temp}/module1.jar" modulename="module1"/>

        <kotlinc src="${test.data}/module2.kt" output="${temp}/module2.jar" modulename="module2">
            <classpath>
                <pathelement path="${temp}/module1.jar"/>
            </classpath>
        </kotlinc>

        <java classname="hello.Module2Kt" fork="true">
            <classpath>
                <pathelement path="${temp}/module1.jar"/>
                <pathelement path="${temp}/module2.jar"/>
                <pathelement location="${kotlin.runtime.jar}"/>
            </classpath>
        </java>
    </target>
</project>
