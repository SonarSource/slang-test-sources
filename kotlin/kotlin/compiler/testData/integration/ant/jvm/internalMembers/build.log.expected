OUT:
Buildfile: [TestData]/build.xml

build:
  [kotlinc] Compiling [[TestData]/module1.kt] => [[Temp]/module1.jar]
  [kotlinc] Compiling [[TestData]/module2.kt] => [[Temp]/module2.jar]
     [java] A_OA_K
     [java] B_OB_K
     [java] C_OC_K

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
